### PR TITLE
Chore: Render the visible thumbnails first

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/lib/VirtualScroller.js [hunk 1]
```diff
@@ -425,6 +425,21 @@ class VirtualScroller {
...
+    /**
+     * Gets the currently visible list items
+     * @return {Array<HTMLElement>} - the list of current visible list items
+     */
+    getVisibleItems() {
+        if (!this.listEl) {
+            return [];
+        }
+
+        return Array.prototype.slice
```
Comment (jeremypress) [inline src/lib/VirtualScroller.js]: iirc you do this a few types, might be worth abstracting the conversion of the listEl into an array?

File: src/lib/VirtualScroller.js [hunk 2]
```diff
@@ -425,6 +425,21 @@ class VirtualScroller {
...
+            .call(this.listEl.children)
+            .filter((itemEl) => this.isVisible(itemEl))
+            .map((itemEl) => itemEl && itemEl.children && itemEl.children[0]);
```
Comment (jeremypress) [inline src/lib/VirtualScroller.js]: Why do we need to look at the first child of the item?

Comment (ConradJChan) [reply src/lib/VirtualScroller.js]: From a `VirtualScroller` perspective, the first child of the `itemEl` will be what the user of the `VirtualScroller` generated, or in this case, the generated thumbnail. Basically, it's stripping away the `VirtualScroller` shell and returning just the items

Comment (jeremypress) [reply src/lib/VirtualScroller.js]: 👍 , just wanted to make sure it wasn't looking at something thumbnail sidebar specific.

**Pull request merged**